IndyAV/control/indyav_control/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ add_message_files(
   DIRECTORY msg
   FILES
   SteeringStamped.msg
-  ThrustStamped.msg
+  VelocityStamped.msg
```

Comment:
I feel like the msg name `VelocityStamped` does not convey the fact that this message is for the commanded rotational velocity of the back wheels in radians per second maybe change it to something like `RevsStamped`